<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chess</class>
 <widget class="QMainWindow" name="chess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>chess</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>650</width>
      <height>650</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="white_timer">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>30</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="black_timer">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>560</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="white_done">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>done</string>
    </property>
   </widget>
   <widget class="QPushButton" name="black_done">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>560</y>
      <width>101</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>done</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1259</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
